### ISSUE TITLE
Homepage

### ISSUE THREAD

**Issue by camball** [camball/camball.io#19]
the sickest homepage u ever seen

## To Do

- [x] Design
  - [x] Something in Papyrus font, lol. – EDIT: moved to new task, checking off in this issue.
  - [x] [Bows](https://emojicombos.com/bow): 🎀 or ⋆𐙚₊˚⊹ ᡣ𐭩 or ˚⟡˖ ࣪
  - [x] Gotta have something with [three.js](https://threejs.org) and [GSAP](https://gsap.com). The homepage should feel very human and like it's real. – EDIT: moved to new task, checking off in this issue.
- [x] Link in my GitHub README bio once done

**Comment by camball** [comment]
Had a weird, accidental branching issue where I accidentally pushed this code's issue straight to main without realising.

My usual set of custom shell commands I use for my branching strategy (`gnrb` ("git new remote branch") and `gcle` "git clean") failed due to a bug where the command would output a very small failure message amongst many lines of text if the branch name I specified already exists. As such, I missed the error message, and didn't realise my local branch didn't switch. This is now not only fixed in my branching flow commands (`gnrb` fails if the branch exists on either local or remote), but I also enabled the `Require a pull request before merging` branch protection rule. I mistakenly had thought I had this enabled on main all along from when I did initial repo setup, which I evidently didn't.

RCA: two separate causes, all fixed, so this never happens again for this repo... and a failure story in my brain that I've learned from.

Didn't discover this even happened until I went to modify a commit message with `git commit --amend`, went to force push, then got blocked by the branch protections I *did* enable (thankfully, lol). Since I am currently the only person working on this repo, I made the executive decision to do a one-time bypass of branch protections and force push to main to correct the mistake. With the new branch protections enabled, I will never need to do that again.
